cmd/run.go

```diff
@@ -153,6 +179,12 @@ func (c *cmdRun) run(cmd *cobra.Command, args []string) (err error) {
 		}()
 	}
 
+	// TODO: Subscribe all initialization processes (outputs, VUs and executors)
+	// to the Init event. This would allow running them concurrently, and they
+	// could be synchronized by waiting for the event processing to complete.
+	// This could later be expanded to also initialize browser processes.
```

Comment:
> Mostly because I would not find it more readable, and I don't see a case where that will be beneficial.

The benefit is explained in the TODO: it would allow running all independent initialization processes concurrently, and waiting for them to be done before starting the test. Currently the outputs are initialized first, then the VUs and executors. If we plan to eventually also do browser initialization at this point, which we mostly agree on, then running these sequentially doesn't make sense.

I agree that events make the code more difficult to follow, but the implementation here is not any different from using a WaitGroup or a channel to synchronize concurrent tasks. It just formalizes it with event names.

But since it's too early to make this decision, and it would need more discussion, I'll remove the TODO.